Sources/SourceKitLSP/TestDiscovery.swift

```diff
@@ -21,6 +21,22 @@ public enum TestStyle {
   public static let swiftTesting = "swift-testing"
 }
 
+public struct AnnotatedTestItem {
+  /// The test item to be annotated
+  public var testItem: TestItem
+
+  /// Whether the `TestItem` is declared in an extension.
```

Comment:
This is no longer whether the test item is *in* an extension but whether it *is* an extension, right?

```suggestion
  /// Whether the `TestItem` is an extension.
```